### DISCUSSION TITLE
Debian 上的青龙通过SSH开启隧道并保持SSH连接才能使隧道在线

### DISCUSSION THREAD

**Discussion by klcb2010** [natfrp/wiki#338] [水深火热]
断开SSH  隧道立即下线

**Comment by fakeboboliu** [comment]
请在文档 repo 中提问时，起码先看一眼 [文档](https://doc.natfrp.com/frpc/service/systemd.html) 吧

**Reply by klcb2010** [reply]
> 请在文档 repo 中提问时，起码先看一眼 [文档](https://doc.natfrp.com/frpc/service/systemd.html) 吧
之前按照这个指导失败了的，今天用ai机器人解决了
